app/models/agents/data_output_agent.rb

```diff
@@ -45,9 +45,14 @@ class DataOutputAgent < Agent
               "_contents": "tag contents (can be an object for nesting)"
             }
 
-        # Ordering events in the output
+        # Ordering events
 
-        #{description_events_order('events in the output')}
+        #{description_events_order('events')}
+
+        DataOutputAgent will select the last `events_to_show` entries of its received events sorted in the order specified by `events_order`, which is defaulted to the event creation time.
+        So, if you have multiple source agents that may create many events in a run, you may want to either increase `events_to_show` to have a larger "window", or specify the `events_order` option to an appropriate value (like `date_published`) so events from various sources are properly mixed in the resulted feed.
+
+        There is also an option `events_list_order` to control the order of events listed in the output, with the same format as `events_order`.  It is defaulted to `#{Utils.jsonify(DEFAULT_EVENTS_ORDER['events_list_order'])}` so the latest entry is listed first.
```

Comment:
Or

"There is also an option `events_list_order` that only controls the order of events listed in the output, without attempting to maintain a total order of received events.  It has the same format as `events_order`."

I'm having trouble verbalizing the difference between these two subtly-different behaviors.